CommandLine/protos/resources/Path.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto2";
+package buffers.resources;
+
+import "options.proto";
+
+message Path {
+  message Point {
+    optional int32 x = 1 [(gmx) = "GMX_SPLIT/0"];
+    optional int32 y = 2 [(gmx) = "GMX_SPLIT/1"];
+    optional int32 speed = 3 [(gmx) = "GMX_SPLIT/2"];
+  }
+
+  optional string name = 1;
+  optional int32 id = 2 [(gmx) = "GMX_DEPRECATED"];
+
+  optional string background_room_name = 3 [(gmx_id) = "backroom"];

```

Comment:
Isn't this group just for the IDE? If we have the ability to extract these, I'd extract them.

Reply:
Yes it is, you don't need this in the engine at all. In GMS2 paths are a subresource of rooms I think.